core/domain/takeout_service.py

```diff
@@ -42,34 +70,26 @@ def export_data_for_user(user_id):
                                 model export policy>
         }
     """
-    models_to_export = [
-        user_models.UserStatsModel,
-        user_models.UserSettingsModel,
-        user_models.UserSubscriptionsModel,
-        user_models.UserSkillMasteryModel,
-        user_models.UserContributionsModel,
-        user_models.ExplorationUserDataModel,
-        user_models.CompletedActivitiesModel,
-        user_models.IncompleteActivitiesModel,
-        user_models.ExpUserLastPlaythroughModel,
-        user_models.LearnerPlaylistModel,
-        user_models.CollectionProgressModel,
-        user_models.StoryProgressModel,
-        feedback_models.GeneralFeedbackThreadModel,
-        feedback_models.GeneralFeedbackMessageModel,
-        collection_models.CollectionRightsModel,
-        suggestion_models.GeneralSuggestionModel,
-        exploration_models.ExplorationRightsModel,
-        email_models.GeneralFeedbackEmailReplyToIdModel
+    all_models = []
+    model_names_list = [
+        model_name
+        for model_name in dir(models.NAMES)
```

Comment:
Can probably put this part at end of previous line.

Reply:
Done